dev-libs/libuv/libuv-1.21.0.recipe

```diff
@@ -44,10 +46,17 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+defineDebugInfoPackage libuv$secondaryArchSuffix \
+	"$libDir"/libuv.so.$libVersion
+
 BUILD()
 {
 	./autogen.sh
 
+	export CFLAGS="-DB_USE_POSITIVE_POSIX_ERRORS -D_BSD_SOURCE"
+	export CXXFLAGS=$CFLAGS
```

Comment:
```diff
-	export CFLAGS="-DB_USE_POSITIVE_POSIX_ERRORS -D_BSD_SOURCE"
-	export CXXFLAGS=$CFLAGS
+	export CPPFLAGS="-DB_USE_POSITIVE_POSIX_ERRORS -D_BSD_SOURCE"
```

Maybe these should be added into the patch as well. From what I've seen, `-D_BSD_SOURCE` was added.